articles/portal/ptl-ref-release-notes.md

```diff
@@ -19,6 +19,16 @@ toc_mdlink: ptl-ref-release-notes.md
 
 This article provides a list of completed Portal releases.
 
+## Release 2.4.0
+
+Date: 28th October 2020
+
```

Comment:
@ccouzens in line with other updates, can you add "Feature update" please.

Reply:
Done